pkgs/by-name/mc/mcdreforged/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+  testers,
+  mcdreforged,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "mcdreforged";
+  version = "2.12.3";
```

Comment:
You should specific pyproject = true or format if that doesn't work

Reply:
Hi Sandro,

I wish I could understand your approach.

Between opening PR and leaving them without maintenance for months and this pattern of reviewing PRs after they are merged is becoming common, and honestly, I don't see the point. 

First, you're wasting your time. Since you mentioned on Matrix that you’re "a busy man", I don’t see how this is beneficial for either you or the PR author.

Second, given your expertise with Nix, it would likely be faster and more efficient for you to provide a PR to address the issue directly.

Third, this particular PR had been awaiting review/merge for 3 weeks. Only after it was merged did you provide a review. Do you understand how frustrating this is for contributors?

Reply:
This is also feedback to you, especially in the python package area where you are not aware of all the reviewing points. All the examples in https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md mention pyproject and if we want to make it a hard requirement, it is easier if many packages already contain it.

> Second, given your expertise with Nix, it would likely be faster and more efficient for you to provide a PR to address the issue directly.

Not necessarily, especially when I am reviewing things on the phone where opening a PR is not as easy.

> Third, this particular PR had been awaiting review/merge for 3 weeks. Only after it was merged did you provide a review. Do you understand how frustrating this is for contributors?

It probably came up in the merge/review threads recently, hence I and you picked it up

Reply:
> This is also feedback to you, especially in the python package area where you are not aware of all the reviewing points. All the examples in https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md mention pyproject and if we want to make it a hard requirement, it is easier if many packages already contain it.

Every piece of feedback is good, but I really wish they could be pointed out in time, so the original author of the PR can learn from them and adjust its PR, without frustration.

> Not necessarily, especially when I am reviewing things on the phone where opening a PR is not as easy.

Then maybe it's better to not say anything at all in that case?